keps/sig-network/4444-service-traffic-distribution/README.md

```diff
@@ -346,6 +412,41 @@ NOTE: The fact that EndpointSlice hints are not expected to be implemented by
   kube-controller-manager based implementations to say that we do support the
   standard heuristics.
 
+### Standard Heuristic Implementation (kube-proxy dataplane)
+
+kube-proxy will start with supporting three distinct behaviors based on the value
+configured for `trafficDistribution` (or, more precisely, based on the hints that the
+EndpointSlice controller sets):
+
+#### Default (i.e. `trafficDistribution` is not configured, no EndpointSlice hints)
+
+* **Interpretation:** When `trafficDistribution` is not used, kube-proxy would match
+  it's existing behaviour of having an equal distribution of traffic across
+  endpoints (potentially spanning multiple zones or regions)
+
+* This leverages the existing implementation, requiring no major changes.
+
+#### `PreferClose`
+
+* **Interpretation:** Attempts to route traffic to endpoints within the same zone as
+  the client. If no endpoints are available within the zone, traffic would be
+  routed to other zones.
```

Comment:
I may be nitpicking this: "traffic would be routed to other zones."

I assume this means that traffic is routed to all the other zones (as if PreferClose was disabled), but I also read this as "traffic could be sent to another individual zone".

Is it worth making that sentence a bit more clear?